### PR TITLE
Update tts transcript handling for server impl. Also remove old output stuff and replace with the tts-synced version.

### PR DESCRIPTION
**Pull request by mdepinet**
In light of our Friday conversation, I'm planning on breaking the client briefly while the server and client changes roll out out-of-sync.  (Will have a client PR to use this SDK version after standup)

### CODE REVIEW

File: packages/fixie-web/src/voice.ts [hunk 1]
```diff
@@ -356,20 +351,15 @@ export class VoiceSession {
...
+    } else if (msg.delta) {
       this.outputTranscript += msg.delta;
-      this.onOutputTranscript?.(this.outputTranscript);
-      if (msg.final) {
-        this.outputTranscript = '';
-      }
-    } else {
-      this.onOutputTranscript?.(msg.text);
+    }
+    this.onOutputChange?.(this.outputTranscript, msg.final);
```
Comment (juberti) [inline packages/fixie-web/src/voice.ts:360]: In the case where only `text` is filled in, are we OK with this being undefined?

**Review by juberti — APPROVED** [review]

File: packages/fixie-web/src/voice.ts (hunk 1 above)
Comment (mdepinet) [reply packages/fixie-web/src/voice.ts:360]: I don't understand your question.  What is `this`?  With the exception of onOutputChange, nothing here should be undefined once the server change lands.

Comment (mdepinet) [reply packages/fixie-web/src/voice.ts:360]: Perhaps you're referring to the interim case before the server change lands. In that case, AI.Town doesn't use onOutputChange right now, so it won't make any difference.  (It uses onOutputTranscript, which means it won't show agent transcripts until the server piece lands.)

Comment (mdepinet) [reply packages/fixie-web/src/voice.ts:360]: I'm going to start merging things so I can get everything landed today. If there are issues, then I'll fix them later. :)

**Pull request merged**